src/ui/FullStoragesChanges.js

```diff
@@ -20,24 +20,15 @@ FullStoragesChanges.prototype.render = function () {
   return view
 }
 
-FullStoragesChanges.prototype.hide = function () {
-  this.view.style.display = 'none'
-}
-
-FullStoragesChanges.prototype.show = function () {
-  this.view.style.display = 'block'
-}
-
 FullStoragesChanges.prototype.init = function () {
   var self = this
   this.parent.register('newTraceLoaded', this, function (length) {
     self.panels = []
     self.traceManager.getAddresses(function (error, addresses) {
       if (!error) {
         self.addresses = addresses
-        self.basicPanel.data = ''
-        yo.update(self.view, self.render())
-        self.hide()
+        self.basicPanel.data = {}
```

Comment:
I don't think it is good style to access the data attribute directly here. What about providing an argument for the update function?